packages/sdks/README.md

```diff
@@ -1,3 +1,36 @@
 # Builder.io SDKs
 
 Generated by JSX Lite
+
+# Feature Implementation
+
+Legend:
+
+- ✅: implemented
+- 🏗: currently in progress
+- ⚠️: not-yet implemented
+- N/A : does not apply
+
+| Builder Blocks | Vue | Vue details | React-Native | React-Native details |
+| -------------- | --- | ----------- | ------------ | -------------------- |
+| Columns        | 🏗   |             | ✅           |                      |
+| Text           | ✅  |             | ✅           |                      |
+| Image          | ✅  |             | ✅           |                      |
+| Buttons        | ✅  |             | 🏗            |                      |
+| Video          | ✅  |             | 🏗            |                      |
+| Custom Code    | ✅  |             | ✅           |                      |
+| Section        | ✅  |             | ✅           |                      |
+| Form           | ✅  |             | ✅           |                      |
+| Fragment       | ✅  |             | ✅           |                      |
+| Input          | ✅  |             | ✅           |                      |
+
+| Features                                  | Details                                                  | Vue | Vue details | React-Native | React-Native details |
+| ----------------------------------------- | -------------------------------------------------------- | --- | ----------- | ------------ | -------------------- |
+| A/B Tests                                 | No SSR support                                           | ✅  |             | ✅           |                      |
+| SSR                                       |                                                          | ✅  |             | N/A          |                      |
+| Children for Custom Components            |                                                          | 🏗   |             | 🏗            |                      |
+| Dynamic Data Bindings                     |                                                          | ⚠️  |             | ⚠️           |                      |
+| Animations                                |                                                          | ⚠️  |             | ⚠️           |                      |
+| View Current Draft                        |                                                          | ⚠️  |             | ⚠️           |                      |
+| Symbols                                   |                                                          | ⚠️  |             | ⚠️           |                      |
+| `+ add block` button when starting a page | (for now just drag your first block to the "layers" tab) | ⚠️  |             | ⚠️           |                      |
```

Comment:
@steve8708 Could you please take a look at this and let me know if there is anything you know of that I may have missed? 🙏🏽

Reply:
love it! I think the only wrong thing is we don't support Vue A/B testing the way we will want to. We'll want to do the [SSG supported way](https://www.builder.io/blog/high-performance-no-code#:~:text=static%20generated%20a%2Fb%20testing) we do with React. So I'd just change that to be a no for now and we can circle back on

Also @teleaziz is a good person to ping re: how the SSG supported a/b testing works

Reply:
Thanks for the heads up, noted.